src/App.tsx

```diff
@@ -5,12 +5,14 @@ import ROUTER_CONFIG from './navigation/router-config';
 
 
 import Header from './lib/components/Header/Header';
+import ImportantContainer from './modules/main/components/ImportantContainer';
 
 const App: React.FC  =() => {
   return (
     <Suspense fallback={<div>Загрузка...</div>}>
       <div className="App">
         <Header/>
+        <ImportantContainer />
```

Comment:
should me on the MainView, since it's valid only for main page

Reply:
fixed